ecommerce/products/templates/product_by_type.html

```diff
@@ -2,14 +2,21 @@ <h1>Products by Type</h1>
 
 <h2>All Products of This Type</h2>
 
+<a href="{% url 'home' %}">
+<h3>Home</h3>
```

Comment:
I don't think it is appropriate to use a `h3` for a link. All `h` tags are for headers. Try just putting the text in the `a` tag: `<a href="{% url 'home' %}">Home</a>`